src/logging.c

```diff
@@ -17,10 +17,14 @@ extern "C"
 {
 #endif
 
+#include <string.h>
+
 #include "rcutils/allocator.h"
+#include "rcutils/get_env.h"
 #include "rcutils/logging.h"
 
 bool g_rcutils_logging_initialized = false;
+const char * g_rcutils_logging_output_format_string = "[{severity}] [{name}]: {message} {line_number}";
```

Comment:
For the default I would suggest to leave the `line_number` out. Without the filename it isn't very helpful anyway.